.taskcluster.yml

```diff
@@ -45,14 +45,38 @@ tasks:
                cd bugbug &&
                git checkout ${head_rev} &&
                pip install -r test-requirements.txt &&
-               flake8 &&
-               python infra/version_check.py"
+               flake8"
         metadata:
           name: bugbug lint
           description: bugbug lint
           owner: ${user}@users.noreply.github.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-push"'
+        then:
+          taskId: {$eval: as_slugid("version_check_task")}
```

Comment:
This task should be added as a dependency of the release task.

Reply:
So do you mean to define this task in the release task itself or just adding the dependency like we've done for other tasks?

Reply:
Just the dependency